src/config/dynamic-theme-fixes.config

```diff
@@ -6506,6 +6506,13 @@ IGNORE IMAGE ANALYSIS
 
 ================================
 
+kali.org
+
+INVERT
+#banner-logo
```

Comment:
```suggestion
IGNORE IMAGE ANALYSIS
#banner-logo
```

Reply:
I tried using `IGNORE IMAGE ANALYSIS` But did not work on chromium

Reply:
You need to give it a reload before it takes effect ;)